### PR TITLE
Ghc844+update

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+result
```
Comment (roberth) [inline .gitignore:1]:
```suggestion
result
result-*
```

File: nix/stack/pkgs.nix [hunk 2]
```diff
@@ -2,7 +2,7 @@
   extras = hackage:
     {
       packages = {} // {
```
Comment (roberth) [inline nix/stack/pkgs.nix:4]:
```suggestion
       packages = {
```

File: nix/default.nix [hunk 3]
```diff
@@ -29,6 +26,7 @@ with
           # hie-core-ghc844 = mkHieCore pkgs.haskell.compiler.ghc844;
```
Comment (roberth) [inline nix/default.nix]:
```suggestion
          hie-core-ghc844 = mkHieCore pkgs.haskell.compiler.ghc844;
```

File: nix/stack/pkgs.nix (hunk 2 above)
Comment (domenkozar) [reply nix/stack/pkgs.nix:4]: this file is autogenerated

**Comment by domenkozar** [comment]
still working on this, going to rename to ghcide

**Review by roberth — APPROVED** [review]

**Pull request merged**